### ISSUE TITLE
Jupyter Book documentation site for v0

### ISSUE THREAD

**Issue by weiji14** [Clay-foundation/model#90]
With the initial Jupyter Book site setup completed in #89, we can now start to add more sections/sub-sections to the Table of Contents. The website will be hosted at https://clay-foundation.github.io/model/ until we get a proper domain name.

For v0 (due before Christmas 2023), I'm planning the following structure:

## Getting started
  - [x] Installation (`installation.md`)
## Data Preparation
  - [x] Sampling locations (`data_sampling.md`) #124 
  - [x] Preparing the datacube (`data_datacube.md`) #97
  - [x] Preparing labels for evaluating model performance (`data_labels.md`) #100
## Running the model
  - [ ] Self-supervised pretraining (`model_ssl.md`)
  - [x] Generating embeddings (`model_embeddings.md`) #98
  - [x] Finetuning on labels (`model_finetuning.md`) #99

Note that the markdown files for the 'Data' and 'Model' sections are prefixed with `data_` and `model_` respectively to organize things a bit. Feel free though to suggest changes or edit the above directly.

**Comment by brunosan** [comment]
PR opened to complete v0 documentation on #118

**Comment by brunosan** [comment]
I think this is already live and mostly complete for a first pass.
